README.markdown

```diff
@@ -32,14 +32,23 @@ You can connect to multiple instances of nREPL for different projects, and it
 will use the right one automatically.  ClojureScript support is just as
 seamless with [Piggieback][].
 
+If you're using the new [Clojure CLI][], you can follow the instructions for
+[running cider-nrepl with `clj`][cider-nrepl-via-clj].
+Briefly, `clj -Sdeps '{:deps {cider/cider-nrepl {:mvn/version "0.21.1"} }}'
+-m nrepl.cmdline --middleware "[cider.nrepl/cider-middleware]"` should do the trick.
+The [cider-nrepl][cider-nrepl-via-clj] docs also show you how you can add an alias to
+your user's `~/.clojure/deps.edn` file, letting you more simply run `clj -A:cider-clj`.
+
 Oh, and if you don't have an nREPL connection, installing [salve.vim][]
 lets it fall back to using `java clojure.main` for some of the basics, using a
 class path based on your Leiningen or Boot config.  It's a bit slow, but a
 two-second delay is vastly preferable to being forced out of my flow for a
 single command, in my book.
 
 [cider-nrepl]: https://github.com/clojure-emacs/cider-nrepl
+[cider-nrepl-via-clj]: https://github.com/clojure-emacs/cider-nrepl#via-clj
```

Comment:
I was just looking into setting this up today, and it seems like this link got moved? I believe this is where the info lives now: https://github.com/clojure-emacs/cider-nrepl/blob/master/doc/modules/ROOT/pages/usage.adoc#via-clj

Reply:
I merged before reading this, if it's still the case would someone mind sending another PR?